src/Altinn.Broker.Core/Domain/Enums/FileStatus.cs

```diff
@@ -1,10 +1,16 @@
 namespace Altinn.Broker.Core.Domain.Enums;
 
-public enum FileStatus {
-    None = 0,
-    Initialized = 1,
-    Processing = 2,
-    Ready = 3,
-    Failed = 4,
-    Deleted = 5
-}
+public enum FileStatus
```

Comment:
Please update these in separate PR as agreed; https://github.com/Altinn/altinn-broker/issues/164